docusaurus/sidebars.js

```diff
@@ -582,7 +584,19 @@ const sidebars = {
   ],
 
   userDocsSidebar: [
-    "user-docs/intro",
+    {
```

Comment:
These are the only modified lines that we really need in this PR. (lines 587 to 599, and also lines 689 to 696)
Now that you have disabled Prettier, would it be possible to revert all the single quotes `'` changes you've made to double-quotes `"`? You can simply do a manual search & replace. Otherwise this would create unnecessary "commits pollution" with other PRs. 😅

Reply:
Okay, I'll change all double quotes to single quotes on the sidebars.js file